src/hookEvent.h

```diff
@@ -0,0 +1,32 @@
+// Copyright 2015, Christopher J. Foster and the other displaz contributors.
+// Use of this code is governed by the BSD-style license found in LICENSE.txt
+
+#ifndef DISPLAZ_HOOKEVENT_H_INCLUDED
+#define DISPLAZ_HOOKEVENT_H_INCLUDED
+
+#include <QByteArray>
+#include <QKeySequence>
+#include <QShortcut>
+
+/// Class for registering custom hook events. Implemented via QShortCut,
+/// i.e. for QKeySequences only.
+class hookEvent : public QObject
```

Comment:
Actually, before you go renaming, it looks like `HookEvent` isn't right either in the current design: It's actually the full list of hook events, so it should be called `HookEvents` if nothing else.  Let me mull over the design a little.